src/Integration.Vsix/CFamily/CFamilyStrings.resx

```diff
@@ -182,4 +182,14 @@
   <data name="MSG_PchSaved" xml:space="preserve">
     <value>Refreshing PCH file for {0}. PCH file location: {1}</value>
   </data>
+  <data name="ERROR_CreatingConfig" xml:space="preserve">
+    <value>Error retrieving configuration. File: {0}, Error: {1}</value>
+  </data>
+  <data name="MSG_UnableToCreateConfig" xml:space="preserve">
+    <value>Unable to retrieve the configuration for file '{0}'.
```

Comment:
This message is now logged by the analyzer if a request could not be created.

I've reworded it slightly to include "is part of a supported project type..." since it will now appear for VCX and CMake projects.

Reply:
I suggest that we do a clean up in the end and tidy up the messages, decide where to include the doc link for example -- right now it's in the cmake stuff, but it probably should be here instead.